docs/test-framework-dev-guide.md

```diff
@@ -292,6 +292,22 @@ until it reports a failure.
 
 - `TEST_RUN_UNTIL_FAILURE=true mage integration:single [testName]`
 
+### Running integration tests with local changes in beats (for otel receivers)
+
+If you've made local changes to Beats-OTeL and want to run the Agent's integration tests against those changes, follow these steps:
```

Comment:
I realize this is a backport PR, so probably doesn't make sense to fix this here.
```suggestion
If you've made local changes to Beats-OTel and want to run the Agent's integration tests against those changes, follow these steps:
```

Reply:
Updated in https://github.com/elastic/elastic-agent/pull/8015.